pyproject.toml

```
@@ -91,8 +86,9 @@ omit = ["*/vendored/*"]
 [tool.coverage.report]
 show_missing = true
 
-[tool.ruff.lint]
-extend-select = [
+[tool.ruff]
+extend-exclude = ["src/wheel/vendored"]
+lint.extend-select = [
```

This is my preferred way to organize ruff configs.
```suggestion

[tool.ruff.lint]
extend-select = [
```